### ISSUE TITLE
Error in image.nvim interpreting allow-passthrough as per the README instructions

### ISSUE THREAD

**Issue by pbower** [3rd/image.nvim#145]
Error detected while processing /home/pbower/.config/nvim/init.vim:    
line  624:                                                             
E5108: Error executing lua ...share/nvim/plugged/image.nvim/lua/image/u
tils/logger.lua:54: 20:46:08.004198 [image.nvim] tmux does not have all
ow-passthrough enabled                                                 
stack traceback:                                                       
        [C]: in function 'handler'                                     
        ...share/nvim/plugged/image.nvim/lua/image/utils/logger.lua:54:
 in function 'throw'                                                   
        ...vim/plugged/image.nvim/lua/image/backends/kitty/init.lua:29:
 in function 'setup'                                                   
        .../.local/share/nvim/plugged/image.nvim/lua/image/init.lua:66:
 in function 'setup'                                                   
        [string ":lua"]:1: in main chunk     
        
        
        
        
      However:
      
      set -gq allow-passthrough on
      
      
      is set, enabled and sourced in ~/.tmux.conf
      
      
      Why is this ?
      
      Thanks

**Comment by 3rd** [comment]
It can be a temporary glitch with tmux, try killing the server and see if it happens again.
We check that by running `tmux show -Apv allow-passthrough` and we expect it to output `on`.
Try running the command outside of Neovim, if it's not displaying "on" it's something wrong with your tmux, if it is displaying on it's a bug in image.nvim, but I don't think that's possible.

**Comment by TamaMcGlinn** [comment]
I was on the latest tmux packaged for Ubuntu 22.04, which is 3.2, and got `invalid option: allow-passthrough`. After compiling from source to get tmux 3.4 (which also requires building libevent from src, following the readme of tmux), I was able to get `on`. So I guess there's a minimum version of tmux of either 3.3 or 3.4. I have also verified this is necessary on another machine running Ubuntu 20.04.
